core/chains/evm/logpoller/log_poller.go

```diff
@@ -292,7 +293,12 @@ func (lp *logPoller) getReplayFromBlock(ctx context.Context, requested int64) (i
 func (lp *logPoller) run() {
 	defer close(lp.done)
 	logPollTick := time.After(0)
+	// trigger first backup poller run shortly after first log poller run
+	backupLogPollTick := time.After(100 * time.Millisecond)
+
 	blockPruneTick := time.After(0)
+	var firstUnfinalizedBlockNumber int64
```

Comment:
can be a state var on the lp structure itself and I think firstUnfinalizedBlockNumber is confusing better to call it   lastBackupPollBlock